delensalot/biases/iterbiasesN0N1.py

```diff
@@ -0,0 +1,424 @@
+# Adapted from Julien Carron lenspec script
+
+import os
+import numpy as np
+import hashlib
+import pickle as pk
+from healpy import gauss_beam
+from scipy.interpolate import UnivariateSpline as spl
+from plancklens.helpers import cachers, mpi
+from plancklens import qresp, nhl, utils
+
+#Uses lensitbiases to compute fast N1 
+from lensitbiases import n1_fft
+
+# Requires the full camb python package for the lensed spectra calc.
+from camb.correlations import lensed_cls
+
+def cls2dls(cls):
```

Comment:
This and the inverse is already part of [utils](https://github.com/NextGenCMB/delensalot/blob/main/delensalot/utils.py)